src/lib/common/sol-mainloop-impl-riot.c

```diff
@@ -238,16 +87,16 @@ timex_set_until_next_timeout(timex_t *timex)
 void
 sol_mainloop_impl_run(void)
```

Comment:
@lpereira, he's just following the abstraction already in place. We can however improve it. We can drop this and I take a look at pushing (maybe) just the threading part as implementation specific on each OS? This way we could share more code among POSIX and small OSes mainloops.